JotunnLib/Managers/PrefabManager.cs

```diff
@@ -453,7 +453,8 @@ public static Object GetPrefab(Type type, string name)
                     }
                 }
 
-                if (GetCachedMap(type).TryGetValue(name, out var unityObject))
+                var cacheMap = GetCachedMap(type);
+                if (cacheMap != null && cacheMap.TryGetValue(name, out var unityObject))
```

Comment:
GetCachedMap cannot and should not return null in any case, can you explain what's the reasoning here?

Reply:
I obviously hadn't looked too far into GetCachedMap.  Part of my problem resolving my issue is that Reforged is huge so takes forever to test after making changes.  Additionally it is protected by keymanager, thus throws hard when a debugger is attached.

This PR was simply my attempt to prevent any possible NRE from happening within GetPrefab.

I'll take this change out when updating the PR.